server/internal/app/config/mailer.go

```diff
@@ -0,0 +1,20 @@
+package config
+
+type SendGridConfig struct {
+	Email string
+	Name  string
+	API   string
+}
+
+type SESConfig struct {
+	Email string
+	Name  string
+}
+
+type SMTPConfig struct {
+	Host         string
+	Port         string
+	SMTPUsername string
+	Email        string
+	Password     string
+}
```

Comment:
They are not used. `reearthx/mailer` should be used. You can delete them.

Reply:
Done